lib/tasks/move_profile.rake

```diff
@@ -12,7 +12,11 @@ namespace :moves do
 
     moves.find_each(batch_size: 200).with_index do |move, n|
       puts "#{n}/#{total} moves processed ..." if (n % 200).zero? # Show progression
-      profile = move.person.profiles.order(:updated_at).last # Take the profile that was most recently updated
+
```

Comment:
Can't you use your new person.latest_profile method?

Reply:
good pont!